### PR TITLE
Add Swagger for new constructions in Petclinic

### PR DESCRIPTION
**Pull request by kir-r**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,261 +1,274 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (LikhanovMaxim) [inline pom.xml:2]: add to readme - URL to swagger

File: src/main/java/org/springframework/samples/petclinic/coverage/BasicConstructionsController.java [hunk 2]
```diff
@@ -14,6 +16,9 @@
 @RequestMapping("/coverage")
 class BasicConstructionsController {
 
+    @ApiOperation(value = "Testing if-else statement",
+        notes = "Entered value should be true or false. " +
+            "Method returns its text value.")
```
Comment (LikhanovMaxim) [inline src/main/java/org/springframework/samples/petclinic/coverage/BasicConstructionsController.java]: maybe add desc about coverage / count probes / bytecode?

Comment (LikhanovMaxim) [reply src/main/java/org/springframework/samples/petclinic/coverage/BasicConstructionsController.java]: in another endpoints the same

**Review by LikhanovMaxim — CHANGES_REQUESTED** [review]

**Review by LikhanovMaxim — APPROVED** [review]

File: src/main/java/org/springframework/samples/petclinic/coverage/BasicConstructionsController.java [hunk 3]
```diff
@@ -146,6 +209,7 @@ public String complexCombination(@PathVariable("value") String value) {
 
 }
 
+
```
Comment (ViktoriiaLebedeva) [inline src/main/java/org/springframework/samples/petclinic/coverage/BasicConstructionsController.java]: remove new line

**Review by ViktoriiaLebedeva — APPROVED** [review]

**Pull request merged**